vc_mobile/constants.txt

```diff
@@ -0,0 +1,4 @@
+false=0
```

Comment:
Note to self: this file needs `const..end`  as described in Breaking Changes section
https://github.com/sannybuilder/dev/releases/tag/v3.7.0